utils/validation/src/main.rs

```diff
@@ -0,0 +1,58 @@
+mod generators;
+
+use std::{fs::File, io::BufWriter, path::PathBuf};
+
+use casper_validation::{Fixture, ABI_TEST_FIXTURES};
+use clap::Clap;
+
+#[derive(Clap)]
+#[clap(version = "1.0")]
+struct Opts {
+    #[clap(subcommand)]
+    subcmd: SubCommand,
+}
+
+#[derive(Clap)]
+enum SubCommand {
+    Generate(Generate),
+}
+
+/// Generates example test fixtures from the code.
+///
+/// Do not use with day to day development - for example to fix an error in serialization code by
+/// replacing the fixture with possibly invalid code.
+#[derive(Clap)]
+struct Generate {
+    /// Path to fixtures directory.
+    #[clap(short, long, parse(from_os_str))]
+    output: PathBuf,
+}
+
+impl Generate {
+    fn run(self) -> anyhow::Result<()> {
+        let fixtures = generators::make_abi_test_fixtures()?;
+
+        for Fixture::ABI(file_name, fixture) in fixtures {
+            let output_path = {
+                let mut output_path = self.output.clone();
+                output_path.push(ABI_TEST_FIXTURES);
+                output_path.push(file_name + ".json");
+                output_path
+            };
+
+            let file = File::create(&output_path)?;
```

Comment:
This fails if the `ABI_TEST_FIXTURES` subdirectory doesn't exist.

Since you're using anyhow errors here, it'd be useful to add context to the error.  I got the following error message which didn't help too much in diagnosing what I was doing wrong:
```
Error: No such file or directory (os error 2)
```

Reply:
Addressed